cfy_manager/config.py

```diff
@@ -113,9 +122,26 @@ def dump_config(self):
                 )
             )
 
-    def load_config(self):
+    def load_config(self, config_files=None):
         self._load_defaults_config()
-        self._load_user_config()
+        for config_file in config_files or [CONFIG_FILE_NAME]:
+            config_file_path = self._sanitized_config_path(config_file)
+            if config_file_path:
+                logger.info('Loading configuration from '
+                            '{0}'.format(config_file_path))
```

Comment:
```suggestion
                logger.info('Loading configuration from %s',
                            config_file_path)
```
the usual comment about %s-vs-str.format in logger calls, we're starting to get used to using the correct way
(ditto for all other logger+format calls)

Reply:
sure, I'm still not used to this syntax, thanks